builder/tests/builder/src/main/java/io/helidon/builder/test/testsubjects/MyConfigBeanBlueprint.java

```diff
@@ -52,7 +52,7 @@ interface MyConfigBeanBlueprint {
      */
     @ConfiguredOption(key = "port", value = "8080", allowedValues = {
             @ConfiguredValue(value = "8080", description = "t1"),
-            @ConfiguredValue(value = "false", description = "t2")})
+            @ConfiguredValue(value = "80", description = "t2")})
```

Comment:
what is really needed here is a range of good values.  I think ultimately for this we need to refer to a method or class to instantiate to perform dynamic validations for complex cases.

Reply:
`false` is never a good value for an integer port. Not sure what you mean by this. If you need better validation, you can always have a custom decorator